launchable/utils/click.py

```diff
@@ -48,6 +48,52 @@ def convert(self, value: str, param, ctx):
             value), param, ctx)
 
 
+class KeyValueType(click.Option):
+    def __init__(self, *args, **kwargs):
+        super(KeyValueType, self).__init__(*args, **kwargs)
+        self._previous_parser_process = None
+        self._key_value_parser = None
+
+    def add_to_parser(self, parser, ctx):
+        def parser_process(value, state):
+            errorMessage = "Expected key-value like --option kye=value or --option key value. but got '{}'"
```

Comment:
This variable looks static. I think it should be a constant variable or a class variable.

Reply:
I fixed https://github.com/launchableinc/cli/pull/191/commits/5d45cfab687cfb94130a1465637a79885eb41f43. you mean it, right?